package/MDAnalysis/topology/TOPParser.py

```diff
@@ -221,17 +223,17 @@ def parse(self):
         attrs['segids'] = Segids(np.array(['SYSTEM'], dtype=object))
 
         top = Topology(n_atoms, n_res, 1,
-                       attrs=attrs.values(),
+                       attrs=list(attrs.values()),
```

Comment:
So this is nitpicky, but the code in `Topology.__init__` just iterates over `attrs`, so we can just pass the generator this returns